### ISSUE TITLE
Ability to configure a warning on SQL execution number

### ISSUE THREAD

**Issue by jeanbisutti** [quick-perf/quickperf-live#8]
This issue aims to display a warning if the SQL execution number is greater than a configured threshold.

A ```quickperf.database.sql.execution.detected``` boolean property will enable the detection.  

A ```quickperf.database.sql.execution.threshold``` number property will allow configuring the threshold value.

The configuration could be displayed and updated after the application startup with the help of MBeans: 

```
QuickPerf
  -- Database
      -- Operations
           -- boolean isSqlExecutionDetected()
           -- void setSqlExecutionDetected(boolean)
           -- int getSqlExecutionThreshold()
           -- void setSqlExecutionThreshold(int)
```

**Comment by debbeca** [comment]
Hi jean 
Im intrested on working on this one.
what is the default threshold of sql executions ? 
is 20 ok ? or is there any other reference ?

**Comment by jeanbisutti** [comment]
Hi Walid,
Great!
Perhaps 10 for the default value?
Don't hesitate if you need hints for the development.

**Comment by debbeca** [comment]
Good. Am on it!

**Comment by jeanbisutti** [comment]
Great! You may have to build [Quick SQL test data](https://github.com/quick-perf/quick-sql-test-data) before (if you don't have Docker installed, you can skip the tests)

**Comment by jeanbisutti** [comment]
_Quick SQL test data_ is used for the [test generation](https://github.com/quick-perf/quickperf-live/blob/main/doc/test_generation.md)

**Comment by jeanbisutti** [comment]
Fixed with #21
